docs/development/local_setup.md

```diff
@@ -209,58 +187,15 @@ To tear down the local Garden cluster and remove the Docker containers, simply r
 make local-garden-down
 ```
 
-**Using minikube**
-
-Alternatively, spin up a cluster with minikube with this command:
-
-```bash
-minikube start --embed-certs #  `--embed-certs` can be omitted if minikube has already been set to create self-contained kubeconfig files.
-😄  minikube v1.8.2 on Darwin 10.15.3
-🔥  Creating virtualbox VM (CPUs=2, Memory=2048MB, Disk=20000MB) ...
-[...]
-🏄  Done! Thank you for using minikube!
-```
-
-**Using a remote cluster as Garden cluster**
-
-For some testing scenarios, you may want to use a remote cluster instead of a local one as your Garden cluster. 
-To do this, you can use the "remote Garden cluster setup" residing in `hack/remote-garden`. 
-To avoid mistakes, the remote cluster must have a `garden` namespace labeled with `gardener.cloud/purpose=remote-garden`. 
-You must create the `garden` namespace and label it manually before running `make remote-garden-up` as described below.
-
-Use the provided `Makefile` rules to bootstrap your remote Garden:
-
-```bash
-export KUBECONFIG=<path to kubeconfig>
-make remote-garden-up
-[...]
-# Start gardener etcd used to store gardener resources (e.g., seeds, shoots)
-Starting gardener-dev-remote gardener-etcd cluster!
-[...]
-# Open tunnels for accessing local gardener components from the remote cluster
-[...]
-```
-
-This will start an `etcd` instance for the `gardener-apiserver` as a Docker container, and open tunnels for accessing local gardener components from the remote cluster.
-
-To close the tunnels and remove the locally-running Docker containers, run:
-
-```bash
-make remote-garden-down
-```
-
-> Note: The minimum K8S version of the remote cluster that can be used as Garden cluster is `1.19.x`.
-
-> ⚠️ Please be aware that in the remote garden setup all Gardener components run with administrative permissions, i.e., there is no fine-grained access control via RBAC (as opposed to productive installations of Gardener).
```

Comment:
Why do you remove the remote setup?

Reply:
re-introduced the local and remote setups as alternatives with a new change. WDYT?